src/files.ts

```diff
@@ -195,13 +195,13 @@ export async function pushFiles() {
         }, {}, (error: any) => {
           spinner.stop(true);
           if (error) {
-            logError(null, LOG.PUSH_FAILURE);
+            console.error(LOG.PUSH_FAILURE);
```

Comment:
Can we remove this change?
If we're going to change `logError`, we should do it everywhere and in a different PR.

Reply:
Hey, so we're not changing it everywhere, just in this specific case:

```
          if (error) {
            console.error(LOG.PUSH_FAILURE);
            error.errors.map((err: any) => {
              console.error(err.message);
            });
            console.error(LOG.FILES_TO_PUSH);
            nonIgnoredFilePaths.map((filePath: string) => {
              console.error(`└─ ${filePath}`);
            });
            process.exit(1);
```
Since `logError` ends with `process.exit(1)` we never reach the next logs, so if there is an error on this command, it only shows:

`Push failed. Errors:`

And no errors, because it exits. I think this is an isolated enough problem that we don't need any more changes. One thing we could do is have `logError` accept a boolean of whether or not to exit.

Reply:
OK. Please comment this reason in the code.
We can change the `logError` logic later so we exit properly everywhere.

Reply:
Done.